yarn.lock

```diff
@@ -10027,6 +10044,14 @@ react-virtualized@^9.18.5:
     prop-types "^15.7.2"
     react-lifecycles-compat "^3.0.4"
 
+react-window@^1.8.7:
```

Comment:
we can delete this from the UI

Reply:
looks like it's not in package json, i'm not sure why clockface dependency is being added to UI's yarn lock.

Reply:
@ChitlangeSahas Looks like we can't remove it - Updated with `yarn install` and ran a `yarn why`, and it looks like it has to stay as a dependency of clockface:

![Screen Shot 2022-07-07 at 11 09 54 AM](https://user-images.githubusercontent.com/91283923/177808078-12a5ddc0-19b1-45d4-b442-b14871171d33.png)